Incompatible with Node 17

**Describe the bug**
When running `npm i` I am prevented from installing.

**To Reproduce**
Steps to reproduce the behavior:
1. Update node to version 17
2. Attempt package installation

**Expected behavior**
I expect installation to work.

**Screenshots**
```log
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: rethinkdb-ts@2.4.15
npm ERR! notsup Not compatible with your version of node/npm: rethinkdb-ts@2.4.15
npm ERR! notsup Required: {"node":">=12.x <=16.x"}
npm ERR! notsup Actual:   {"npm":"8.1.0","node":"v17.0.1"}
```

**System info**
 - OS: macOS Monterey 12.0.1
 - RethinkDB Version: 2.4.1

**Additional context**
I don't think CPU is a factor but I am using an M1 Max.

EDIT: Installing node version `16.13.0` via nvm works.

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "test": "test"
   },
   "engines": {
-    "node": ">=12.x <=16.x"
+    "node": ">=12.x <=17.x"
   },
   "scripts": {
     "prepare": "husky install && tsc",

